test-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/RestRequestDelegate.groovy

```diff
@@ -137,6 +147,12 @@ class RestRequestDelegate {
             return new XPathAssertionDelegate(xpath, assertionBuilders)
         }
 
-    }
+        XQueryAssertionDelegate xQuery(String xQuery) {
+            return new XQueryAssertionDelegate(xQuery, assertionBuilders)
+        }
 
+        ResponseSlaAssertionDelegate responseSLA(int maxResponseTime) {
```

Comment:
I have changed it to 'maxResponseTime()', I think that's better than just responseTime.